code/src/components/TaskInput.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState }  from 'react';
+import { useDispatch } from 'react-redux';
+import uniqid from 'uniqid';
+
+import tasks from 'reducers/tasks';
+
+
+const TaskInput = () => {
+    const [inputValue, setInputValue] = useState('');
+
+    const dispatch = useDispatch();
+
+    const onFormSubmit = (event) => {
+        event.preventDefault();
+
+        const newTask = {
+            id: uniqid(),
+            text: inputValue,
+            complete: false,
```

Comment:
Here you can put `required`. Then it won't be able to go next step with empty task.